### PR TITLE
feat: Exclude albums option & exclude videos

### PR DESCRIPTION
**Pull request by JW-CH**
#96 #109

### CODE REVIEW

**Comment by JW-CH** [comment]
TODO: add new settings to settingspage-ui

File: ImmichFrame/Helpers/AssetHelper.cs [hunk 1]
```diff
@@ -145,32 +156,69 @@ private async Task<IEnumerable<AssetResponseDto>> GetMemoryAssets()
...
+            }
+        }
 
-                client.UseApiKey(settings.ApiKey);
-                foreach (var albumId in settings.Albums!)
-                {
-                    try
-                    {
-                        var albumInfo = await immichApi.GetAlbumInfoAsync(albumId, null, null);
+        private async Task<IEnumerable<AssetResponseDto>> GetAlbumAssets(Guid albumId)
```
Comment (3rob3) [inline ImmichFrame/Helpers/AssetHelper.cs:174]: Is this being called/used? I'm not seeing it. If it's not can we combine the other two GetAlbumAssets? I had a hard time following the flow, and would be more in line with the other functions.

File: ImmichFrame/Helpers/AssetHelper.cs [hunk 2]
```diff
@@ -106,8 +106,19 @@ public async Task DeleteAndCreateImmichFrameAlbum()
             }
 
             if (assetsAdded)
+            {
+                // Exclude videos
+                    list = list.Where(x => x.Type != AssetTypeEnum.VIDEO);
```
Comment (3rob3) [inline ImmichFrame/Helpers/AssetHelper.cs:111]: Wow, can't believe it was this easy. Nice!

File: ImmichFrame/Helpers/AssetHelper.cs [hunk 3]
```diff
@@ -233,6 +281,17 @@ private async Task<IEnumerable<AssetResponseDto>> GetPeopleAssets()
                     {
                         var asset = randomAssets.First();
 
+                        if (asset.Type == AssetTypeEnum.VIDEO)
```
Comment (3rob3) [inline ImmichFrame/Helpers/AssetHelper.cs:284]: Same, can't believe it was this easy

**Comment by 3rob3** [comment]
closes #96 closes #109

**Pull request merged**